cmd/main.go

```diff
@@ -61,53 +58,53 @@ VERSION:
   ` + Version +
 	`{{ "\n"}}`
 
-func migrate() {
-	// Migrate config file
-	err := migrateConfig()
-	fatalIf(err, "Config migration failed.")
-
-	// Migrate other configs here.
-}
+func newApp() *cli.App {
+	// Collection of minio commands currently supported are.
+	commands := []cli.Command{}
 
-func enableLoggers() {
-	// Enable all loggers here.
-	enableConsoleLogger()
-	enableFileLogger()
-	// Add your logger here.
-}
+	// Collection of minio commands currently supported in a trie tree.
+	commandsTree := trie.NewTrie()
 
-func findClosestCommands(command string) []string {
-	var closestCommands []string
-	for _, value := range commandsTree.PrefixMatch(command) {
-		closestCommands = append(closestCommands, value.(string))
+	// registerCommand registers a cli command.
+	appendCommand := func(command cli.Command) {
+		commands = append(commands, command)
```

Comment:
The name and comment mismatch. Keep the name of the function as is I.e `registerCommand`